src/utils/searchViewHelper.js

```diff
@@ -13,11 +13,12 @@ import {
 const searchViewHelper = {
   searchByRevisionOrEmail(repository, search) {
     const emailMatch = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
-    const longHashMatch = /[a-zA-Z0-9]{40}/;
+    const longHashMatch = /\b[a-f0-9]{40}\b/;
```

Comment:
realized when testing that it was matching to any alphanumeric string up to 40 characters, instead of exactly 40, and revisions contain only hex digits, so I'm including only letters a-f